src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,10 +584,131 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
+
+    /**
+     *Common method to Get the feedback data
+     */
+
+    protected FeedbackSessionAttributes extractFeedbackSessionData() {
+
+        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
+
+        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
+        newSession.setFeedbackSessionName(Sanitizer.sanitizeTitle(
+                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME)));
+
+        // getting the class name from which it's being called
+        String className = new Exception().getStackTrace()[1].getClassName();
+
+        if ("InstructorFeedbackEditSaveAction".equals(className)) {
```

Comment:
@thenaesh 
>You can override this method (since you've made it protected, after all) in the InstructorFeedbackEditSaveAction class, after defining a more general method here in 

This one i thought of earlier this is not gonna owrk in my case because there difference in the  method definition like one class needs one variable other class doesn't need.So i didn't find any ways to write general definition.

But,
>this.getClass(), which returns an object of type Class<?> which you can compare directly, or use this.getClass().getName() to obtain a string representation. 

It will definately help me.Thanks a lot